CMakeLists.txt

```diff
@@ -134,6 +135,7 @@ add_library(
   src/llvm_eval.cpp
   src/llvm_scatter.h
   src/llvm_scatter.cpp
+  src/llvm_ts.cpp
```

Comment:
Can you add the header file here as well?

Reply:
ditto for the CUDA version. That makes the project nicer for people who use IDEs because the header files are linked in the set of target sources.